src/docker-utils.coffee

```diff
@@ -72,7 +73,7 @@ do ->
 				findSimilarImage(imgDest)
 			.then (imgSrc) ->
 				new Promise (resolve, reject) ->
-					progress request.get("#{config.deltaHost}/api/v2/delta?src=#{imgSrc}&dest=#{imgDest}", timeout: DELTA_REQUEST_TIMEOUT)
+					progress resumable(request, { url: "#{config.deltaHost}/api/v2/delta?src=#{imgSrc}&dest=#{imgDest}", timeout: DELTA_REQUEST_TIMEOUT })
```

Comment:
@abresas I just tested this and it breaks because the statusCode below (line 82) is now 206 instead of 200. Not sure if just testing for 206 should work?